index.rst

```diff
@@ -638,12 +698,92 @@ The following require different procedures to create/modify a configuration
 
 - :ref:`Main and Auxiliary Telescope Pointing Components <section-pointing-component>`
 - :ref:`ATMCS and ATPneumatics <section-atmcs-atpneumatics>`
+- Camera CSCs
+
+CSC Developer Information
+=========================
+
+This section contains information that is primarily of interest to CSC developers.
+However, if issues are being encountered when creating new configurations the information may be pertinent.
+
+Rules Regarding Configuration Definitions and Usage
+---------------------------------------------------
+#.  No default values shall be contained in the configuration schema definition in the CSC repository.
+
+#.  In the configuration repository for the given CSC (e.g `ts_config_attcs <https://github.com/lsst-ts/ts_config_attcs>`_ for the ATDome) there shall be a ``_init.yaml`` file that specifies values that are expected to be common to all sites and/or be relatively static in operations (we intentionally use "_init" instead of "_default").
+
+    - See this :download:`example _init.yaml <_static/_init.yaml>` for the ATSpectrograph CSC.
+    - This file is the first configuration file loaded by the CSC.
+    - Providing the ``_init.yaml`` file (or any file with a ``_`` prefix) to the ``configurationOverride`` parameter will return an error.
```

Comment:
This is not a step and can probably be omitted. 

If you feel it is important please consider making it parenthetica, e.g. reword as "Note: providing the...".

Reply:
This is the developer section, so these comments apply to all CSCs (even the non-salobj ones). I reworded to make it stronger.